integrationtests/miniblocks_test.go

```diff
@@ -0,0 +1,124 @@
+//go:build integrationtests
+
+package integrationtests
+
+import (
+	"context"
+	"testing"
+
+	dataBlock "github.com/multiversx/mx-chain-core-go/data/block"
+	"github.com/multiversx/mx-chain-core-go/marshal"
+	indexerdata "github.com/multiversx/mx-chain-es-indexer-go/process/dataindexer"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIndexMiniBlocksOnSourceAndDestination(t *testing.T) {
+	setLogLevelDebug()
```

Comment:
why is this needed?

Reply:
just for an informative purpose, to see logs with Debug when integration tests are running